File: Src/java/engine-fhir/src/test/java/org/opencds/cqf/cql/engine/fhir/model/CachedDstu3FhirModelResolver.java
package org.opencds.cqf.cql.engine.fhir.model;

import ca.uhn.fhir.context.FhirContext;
import ca.uhn.fhir.context.FhirVersionEnum;

public class CachedDstu3FhirModelResolver extends Dstu3FhirModelResolver {
    public CachedDstu3FhirModelResolver() {
        super(FhirContext.forCached(FhirVersionEnum.DSTU3));
    }
}

File: Src/java/engine-fhir/src/test/kotlin/org/opencds/cqf/cql/engine/fhir/model/CachedDstu3FhirModelResolver.kt
package org.opencds.cqf.cql.engine.fhir.model

import ca.uhn.fhir.context.FhirContext
import ca.uhn.fhir.context.FhirVersionEnum

class CachedDstu3FhirModelResolver :
    Dstu3FhirModelResolver(FhirContext.forCached(FhirVersionEnum.DSTU3))
